ui/src/app/machines/components/MachineHeaderForms/ActionFormWrapper/TagForm/TagFormFields/TagFormFields.tsx

```diff
@@ -19,11 +20,13 @@ type Props = {
 };
 
 export const TagFormFields = ({ machines }: Props): JSX.Element => {
+  const addedTags = useSelectedTags();
   const tags = useSelector(tagSelectors.getManual);
   return (
     <Row>
       <Col size={6} className="col-start-large-4">
         <TagField
+          externalSelectedTags={addedTags}
```

Comment:
This is very confusing language. `addedTags = useSelectedTags()` `externalSelectedTags={addedTags}`

Could it all be `selectedTags`?

Reply:
Yeah, that makes a lot more sense (I copied and pasted and didn't think about the names).